test/e2e/storage/drivers/in_tree.go

```diff
@@ -1183,11 +1195,18 @@ func (h *gcePdDriver) GetClaimSize() string {
 }
 
 func (g *gcePdDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
-	return &testsuites.PerTestConfig{
+	config := &testsuites.PerTestConfig{
 		Driver:    g,
 		Prefix:    "gcepd",
 		Framework: f,
-	}, func() {}
+	}
+	if framework.NodeOSDistroIs("windows") {
+		config.ClientNodeSelector = map[string]string{
+			"beta.kubernetes.io/os": "windows",
```

Comment:
Should also check `kubernetes.io/os": "windows"`
We've promoted the labels to GA